### PR TITLE
regenerate the code from protos: empty and any will be changed

### PR DESCRIPTION
**Pull request by skyao**
# Description

see the issue.

## Issue reference

https://github.com/dapr/dapr/issues/3036

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [ ] Created/updated tests
* [x] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

**Comment by yaron2** [comment]
@skyao the new proto's generated are the "correct" ones.

**Comment by skyao** [comment]
> @skyao the new protos generated in this PR are the "correct" ones.

So what happens to the exist code? Somebody generated code by other version of proto tools? 

How can we avoid this?

**Comment by yaron2** [comment]
> > @skyao the new protos generated in this PR are the "correct" ones.
> 
> So what happens to the exist code? Somebody generated code by other version of proto tools?
> 
> How can we avoid this?

If the current instructions for generation result in what you have in this PR, we are ok and this is probably the result of someone using an older version of protoc. or proto-go.

**Comment by skyao** [comment]
> > > @skyao the new protos generated in this PR are the "correct" ones.
> > 
> > 
> > So what happens to the exist code? Somebody generated code by other version of proto tools?
> > How can we avoid this?
> 
> If the current instructions for generation result in what you have in this PR, we are ok and this is probably the result of someone using an older version of protoc. or proto-go.

we can add some checking logic in make file to checkout the output of following commands:

```
% protoc --version
libprotoc 3.14.0
% protoc-gen-go --version
protoc-gen-go v1.25.0
% protoc-gen-go-grpc --version
protoc-gen-go-grpc 1.1.0
```

if the version is not expected, we can warn and quit the make.

**Comment by yaron2** [comment]
> > > > @skyao the new protos generated in this PR are the "correct" ones.
> > > 
> > > 
> > > So what happens to the exist code? Somebody generated code by other version of proto tools?
> > > How can we avoid this?
> > 
> > 
> > If the current instructions for generation result in what you have in this PR, we are ok and this is probably the result of someone using an older version of protoc. or proto-go.
> 
> we can add some checking logic in make file to checkout the output of following commands:
> 
> ```
> % protoc --version
> libprotoc 3.14.0
> % protoc-gen-go --version
> protoc-gen-go v1.25.0
> % protoc-gen-go-grpc --version
> protoc-gen-go-grpc 1.1.0
> ```
> 
> if the version is not expected, we can warn and quit the make.

That's a good idea. /cc @artursouza

**Comment by skyao** [comment]
@yaron2 please merge this PR first to avoid code conflict. If we agree to do some improvement in the make file, we can PR later.

**Review by artursouza — APPROVED** [review]

**Comment by artursouza** [comment]
go mod tidy, please.

I am OK with automating the proto generation with the warning done.

In addition, please, add a step to fail build if the proto output causes changes, similar to how we do for go mod tidy.

File: go.sum [hunk 1]
```diff
@@ -363,6 +363,7 @@ github.com/evanphx/json-patch v4.9.0+incompatible h1:kLcOMZeuLAJvL2BPWLMIj5oaZQo
 github.com/evanphx/json-patch v4.9.0+incompatible/go.mod h1:50XU6AFN0ol/bzJsmQLiYLvXMP4fmwYFNcr97nuDLSk=
 github.com/fasthttp-contrib/sessions v0.0.0-20160905201309-74f6ac73d5d5 h1:M4CVMQ5ueVmGZAtkW2bsO+ftesCYpfxl27JTqtzKBzE=
 github.com/fasthttp-contrib/sessions v0.0.0-20160905201309-74f6ac73d5d5/go.mod h1:MQXNGeXkpojWTxbN7vXoE3f7EmlA11MlJbsrJpVBINA=
+github.com/fasthttp-contrib/websocket v0.0.0-20160511215533-1f3b11f56072 h1:DddqAaWDpywytcG8w/qoQ5sAN8X12d3Z3koB0C3Rxsc=
```
Comment (yaron2) [inline go.sum:366]: Needs `go mod tidy`

Comment (skyao) [reply go.sum:363]: I have update the code and run `go mod tidy`, but no file is changed.

Comment (yaron2) [reply go.sum:363]: The build is failing because of `go.mod` changes. Did you make sure you're using Go 1.16?

Comment (skyao) [reply go.sum]: Yes, I found that my golang is 1.15, after I upgrade to 1.16 the result of go mod t tidy is different to 1.15.

**Comment by skyao** [comment]
@yaron2  @artursouza please help to review and merge this PR to make main branch code "correct".

To improve the make build file and to avoid similar issue, I submit a new PR: https://github.com/dapr/dapr/pull/3044

**Comment by skyao** [comment]
@yaron2  Don't forget to merge this PR.

**Review by yaron2 — APPROVED** [review]

**Pull request merged**